manifest.php

```diff
@@ -38,12 +38,12 @@
     'label' => 'Item core extension',
     'description' => 'TAO Items extension',
     'license' => 'GPL-2.0',
-    'version' => '10.8.4',
+    'version' => '10.8.5',
```

Comment:
SImilarly to [the other PR](https://github.com/oat-sa/extension-tao-itemqti/pull/1499#discussion_r462269504), I'd argue we should bump a major version, but I'd like to see other team member's opinion as well.